### ISSUE TITLE
The Flying Saucer disappears immediately after touching the ground

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#57]
```
What steps will reproduce the problem?
1. Use the saucer
2. Land with it in a normal way (so you dont deal damage)

What is the expected output? What do you see instead?
So my problem is that even if I have some more fuel, it automatically 
disappears when I touch the ground (safely). Or is it intended?

And it also disappears when you fly with it into the water and sink. It could 
be a funny enhancement if the Saucer would remain while sinking (as with the 
bird that flies away in this case)

What version of the product are you using? On what operating system?
0.9.13, Win7

```

-----
Original issue reported on code.google.com by `joship...@gmail.com` on 17 Sep 2010 at 6:48

**Comment by GoogleCodeExporter** [comment]
```
It's intended to go if you stay on the ground for too long (there's a short 
delay where you might take off again). Same goes for Birdy.

I'll keep this open as an enhancement as the drowning idea sounds interesting.
```

-----
Original comment by `mario.li...@gmail.com` on 18 Sep 2010 at 9:51
* Changed state: **Accepted**
* Added labels: **Type-Enhancement**
* Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Yeah, before we cancelled the saucer on touching the ground, knocking hedgehogs 
with it was waaaay too easy.  Also, you could just hang out on saucer, and it 
looked odd sitting on the ground in general.

Dunno about keeping the saucer visible while drowning.  We don't do it for 
anything else (rope, parachute, any formerly active weapon like portal gun).  
Also we'd probably then should keep it visible when tumbling after a fall 
(making it rotate and such along w/ the hog).

Not sure if that's really a good idea.  

Anyway, if this is just an enhancement to drowning, probably should change the 
title at least.

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Sep 2010 at 7:27

**Comment by GoogleCodeExporter** [comment]
```
I see your point, but with most of the other formerly active weapons, it is 
easily imaginable that you dont sink with it (e.g with a parachute, that will 
probably float and not sink), but its just looks a little odd when you are 
flying in(!) a saucer and then it just disappears. of course its just a small 
matter, but looks funny and i guess not so hard to program. 
```

-----
Original comment by `joship...@gmail.com` on 21 Sep 2010 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
Saucer in 0.9.14 now only stops responding if it runs out of fuel.
Also, it is capable of restricted underwater movement.

So, you see a saucer sink/crash just fine.

```

-----
Original comment by `kyberneticist@gmail.com` on 27 Sep 2010 at 2:42
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
cool!:)
```

-----
Original comment by `joship...@gmail.com` on 27 Sep 2010 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
Reopened. In 0.9.17, the saucer disappears after ~0.5 seconds on ground.
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 8:02
* Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
IMO the reopening of this was inappropriate.

The current behaviour on ground touches is deliberate and a WONTFIX.

The only part of the bug that was addressed was the complaint that the saucer 
"vanishes" after running out of fuel.

Specifically, comment #3 about saucer vanishing looking odd.

That's why the bug type was changed to Enhancement.

```

-----
Original comment by `kyberneticist@gmail.com` on 29 Nov 2011 at 3:47
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
To elaborate.
2 parts to bug
1) Saucer vanishes on ground touch
- deliberately added. early saucer did not have it. prevents OP knocks and odd 
saucer appearance, WONTFIX
2) Saucer vanishes when out of fuel
- did indeed look odd, FIXED

```

-----
Original comment by `kyberneticist@gmail.com` on 29 Nov 2011 at 3:54

**Comment by GoogleCodeExporter** [comment]
```
As kyberneticist wrote, this bug has two parts. The first (saucer vanishes on 
ground touch) is actually a feature, the second is fixed.
```

-----
Original comment by `bmarti...@gmail.com` on 26 Nov 2012 at 6:55
* Changed state: **Verified**
